build.sbt

```diff
@@ -70,6 +70,7 @@ val configDependencies = Seq(
 
 val baseDependencies = Seq(
   "org.typelevel" %% "cats-effect" % "3.4.3",
+  "com.github.cb372" %% "cats-retry" % "3.1.0",
```

Comment:
nit: is this change related to the commit message?

Reply:
I was removed in later commits, so it's not necessary. I removed it.